app/src/main/kotlin/com/wire/android/notification/NotificationConstants.kt

```diff
@@ -16,6 +16,9 @@ object NotificationConstants {
     const val MESSAGE_GROUP_KEY = "wire_reloaded_notification_group"
     const val KEY_TEXT_REPLY = "key_text_notification_reply"
 
+    const val OTHER_CHANNEL_ID = "com.wire.android.other_channel"
+    const val OTHER_CHANNEL_NAME = "Others Channel"
```

Comment:
Instead of `Others`, can we name it something like `Message Synchronization`? These will be visible to users, so we should be transparent about what is going on.

Also, if you could add a `TODO` to this `NotificationConstants` it would be great:
These constants should not be constants.
The channel names should be localisable, we should be able to translate them.

So this `NotificationConstants` must be refactored to receive `context` and get String from resources insteda of just having them hardcoded there.

We should be able to inject an instance of `NotificationConstants` where we need it.